check-module-version/README.md

```diff
@@ -0,0 +1,43 @@
+# GitHub Action: Check module version
+
+This action checks the equality of the version from Lua 
+module's code and the repository tag that had triggered the pipeline.
+This CI action works only on tag push event.
+
+## Parameters
+
+- `module-name` — the name of the Lua module.
+  The name format as when using require() in tarantool.
+- `version-pre-extraction-hook` — the string with Lua code.
+  Executed before extracting the version value from _VERSION variable.
+  The hook code should not output to STDERR or STDOUT.
+
+## Example workflow:
+
+```yml
+name: packaging
+on: [...]
+
+jobs:
+  version-check:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check module version
+        # We need this step to run only on push with tag.
```

Comment:
We need to run this step only on push with tag.

Reply:
Done